### PR TITLE
Updates for future FGRL XR

### PR DESCRIPTION
**Pull request by PhillipLucas**
Updated document for forthcoming FGRL XR

### CODE REVIEW

File: windows-apps-src/xbox-live/using-xbox-live/best-practices/fine-grained-rate-limiting.md [hunk 1]
```diff
@@ -124,17 +150,19 @@ Yes, more services and new services can and will be creating limits. Though just
 
 ### When will these changes take effect?
 
-You will notice limits in dev/cert sandboxes starting in April. In May you can expect to see limits in RETAIL
+Rate limites have been enforced since **May 2016**.  As of **April 2018**, titles exceeding the specified sustained limits by 10x or more will not pass the Xbox Certification process.
```
Comment (KevinAsgari) [inline windows-apps-src/xbox-live/using-xbox-live/best-practices/fine-grained-rate-limiting.md]: Typo: "limites"

File: windows-apps-src/xbox-live/using-xbox-live/best-practices/fine-grained-rate-limiting.md [hunk 2]
```diff
@@ -124,17 +150,19 @@ Yes, more services and new services can and will be creating limits. Though just
...
 
 ### What if we can’t adhere to the limits?
 
 Please see the [Xbox Live Best Practices](best-practices-for-calling-xbox-live.md) and ensure you are following these steps.  also consider using the [social manager](../../social-platform/intro-to-social-manager.md) if you are being rate limited with any of the social services.
 
 If after following these steps, you are still unable to remain under the limits, please reach out to your Developer Account Manager.
 
+**NOTE: Titles at or above 10x the specified sustiain limit will not be allowed to pass cert after April 2018**.  For example, if the sustained limit at which FGRL takes effect is set to 300 calls in 300 seconds as specified in the table above, titles at or above 3000 calls in 300 seconds will fail certification.
```
Comment (KevinAsgari) [inline windows-apps-src/xbox-live/using-xbox-live/best-practices/fine-grained-rate-limiting.md]: Typo: "sustiain"

**Review by KevinAsgari — COMMENTED** [review]
Some minor typos

**Comment by mikematteson** [comment]
@KevinAsgari : PR **passed** the basic acceptance checklist. Please evaluate further for technical accuracy and consistency with other Xbox Live documentation.

File: windows-apps-src/xbox-live/using-xbox-live/best-practices/fine-grained-rate-limiting.md [hunk 3]
```diff
@@ -90,30 +90,52 @@ When the associated user and title count is at or above either the burst or sust
...
-| Achievements Read         | 100                                             | 100                                                |
-| Leaderboards Read         | 30                                              | 100                                                |
-| Profile                   | 10                                              | 30                                                 |
-| People                    | 10                                              | 30                                                 |
-| Privacy                   | 10                                              | 30                                                 |
-| Presence                  | Read 10, Write 3                                | Read 100, Write 30                                 |
+The following services will have implemented FGRL limits, with enforcement of these limits in place since **May 2016**. To reiterate, these limits will be the same across all sandboxes and titles. **Any title that is was published via XDP or Dev Center and shipped prior to May 2016 will be considered Legacy and therefore exempted.**
```
Comment (KevinAsgari) [inline windows-apps-src/xbox-live/using-xbox-live/best-practices/fine-grained-rate-limiting.md]: Change "will have implemented" to "have implemented"

Comment (KevinAsgari) [inline windows-apps-src/xbox-live/using-xbox-live/best-practices/fine-grained-rate-limiting.md]: Typo: "Any title that is was" remove the extraneous "is".

Also, change "will be considered Legacy" to "is considered legacy"

File: windows-apps-src/xbox-live/using-xbox-live/best-practices/fine-grained-rate-limiting.md [hunk 4]
```diff
@@ -124,17 +146,19 @@ Yes, more services and new services can and will be creating limits. Though just
...
 Please see the [Xbox Live Best Practices](best-practices-for-calling-xbox-live.md) and ensure you are following these steps.  also consider using the [social manager](../../social-platform/intro-to-social-manager.md) if you are being rate limited with any of the social services.
 
 If after following these steps, you are still unable to remain under the limits, please reach out to your Developer Account Manager.
 
+**NOTE: Titles at or above 10x the specified sustain limit will not be allowed to pass cert after April 2018**.  For example, if the sustained limit at which FGRL takes effect is set to 300 calls in 300 seconds as specified in the table above, titles at or above 3000 calls in 300 seconds will fail certification.
+
 ### What about my existing Title?
 
-Any title in XDP or Dev Center before limits in RETAIL are considered Legacy and exempt
+Any title in XDP or Dev Center before limits in RETAIL are considered Legacy are exempt.
```
Comment (KevinAsgari) [inline windows-apps-src/xbox-live/using-xbox-live/best-practices/fine-grained-rate-limiting.md]: Change to "are considered legacy and are exempt."

**Review by KevinAsgari — APPROVED** [review]

**Pull request merged**